trunk/NDHMS/arc/macros.mpp.intel.cray_xc

```diff
@@ -1,32 +1,3 @@
-## If you have multiple mpi biulds on a single machine
```

Comment:
Do we have info on Hydro-c1 build specifics elsewhere?

I don't usually run the model over there and don't know what the current situation is, but I know it's been a barrier for some folks in the past.  Probably better for an internal site though since it's NCAR specific.

Reply:
I agree that these hydro-c1 notes should be somewhere else and not cluttering up a file most people won't even see. Maybe a GitHub repo Wiki page, or other shared doc?

Reply:
I've had success using GitHub Wiki pages. I think it's a very basic version of MarkDown (limited extensions). The navigation is kind of clunky but a shared doc may have worse navigation. I'm interested in what you decide because the documentation for coupling configurations needs to be cleaned up.